Doc/conf.py

```diff
@@ -94,7 +99,7 @@
 # Ignore any .rst files in the includes/ directory;
 # they're embedded in pages but not rendered individually.
 # Ignore any .rst files in the venv/ directory.
-exclude_patterns = ['includes/*.rst', 'venv/*', 'README.rst']
+exclude_patterns = ['includes/*.rst', 'venv/*', 'venv-3.11/*', 'README.rst']
```

Comment:
Where does `venv-3.11` come from?

Reply:
It doesn't ([`d4bc22c` (#125276)](https://github.com/python/cpython/pull/125276/commits/d4bc22cc4c043925f85a6fd1a1ded834bfc35d5a))